File: backup/com/edge/core/api/APIConstants.java
package com.edge.core.api;

public interface APIConstants {

    // UserController

    String URL_CREATE_USER = "/api/unsecured/core/createUser";
    String URL_CREATE_ADMIN = "/api/secured/core/createAdmin";

    String URL_RESET_PASSWORD = "/api/unsecured/core/resetPassword";
    String URL_LOGIN_USER = "/api/unsecured/core/getLoggedInUser";
    String URL_LOGIN_ROLE = "/api/secured/core/getLoggedInRole";
    String URL_AUTH_FAILED = "/api/unsecured/core/authFailed";

    String URL_VERIFICATION_CODE = "/api/unsecured/core/sendVerificationCode";

}

File: frontend/modules/api/APIConstants.js

BASE_URI = "http://localhost:8080";

// UserController

URL_CREATE_USER = BASE_URI + "/api/unsecured/core/createUser";
URL_CREATE_ADMIN = BASE_URI + "/api/secured/core/createAdmin";

URL_RESET_PASSWORD = BASE_URI + "/api/unsecured/core/resetPassword";
URL_LOGIN_USER = BASE_URI + "/api/unsecured/core/getLoggedInUser";
URL_LOGIN_ROLE = BASE_URI + "/api/secured/core/getLoggedInRole";
URL_AUTH_FAILED = BASE_URI + "/api/unsecured/core/authFailed";

URL_VERIFICATION_CODE = BASE_URI + "/api/unsecured/core/sendVerificationCode";


